docs/pages/deploy-a-cluster/teleport-enterprise/getting-started.mdx

```diff
@@ -1,465 +1,489 @@
 
 Signup token has been created and is valid for 1 hours. Share this URL with the user:
 https://auth.example.com:3080/web/newuser/22e3acb6a0c2cde22f13bdc879ff9d2a
 ```
 
-Share the generated sign-up URL with Joe and let him pick a password and configure
-the second factor authentication. We recommend [Google Authenticator](https://en.wikipedia.org/wiki/Google_Authenticator)
-which is available for both Android and iPhone.
+Navigate to the link displayed in your terminal, pick a password, and configure
+second factor authentication.
 
-## Assigning Roles
+### Log in as your new user
 
-To update user's roles, dump the user resource into a file:
+`tsh` is our client tool. It helps you log in to Teleport clusters and obtain
+short-lived credentials. It can also be used to list resources registered with
+Teleport, such as servers, applications, and Kubernetes clusters.
 
-```code
-$ sudo tctl get users/joe > joe.yaml
-```
+Install `tsh` on your local machine:
 
-Edit the YAML file and update the "roles" array.
-Then, re-insert it back:
+<Tabs>
 
-```code
-$ sudo tctl create -f joe.yaml
-```
+<TabItem label="macOS">
 
-## Logging In
+Run the follwoing commands to download and run the Teleport installer:
 
-Joe now has a local account on a Teleport cluster. The local account is good for
-administrative purposes but regular users of Teleport Enterprise should be using
-a Single Sign-On (SSO) mechanism.
+  ```code
+  $ curl -O https://get.gravitational.com/teleport-ent-(=teleport.version=).pkg
+  # Installs on Macintosh HD
+  $ sudo installer -pkg teleport-ent-(=teleport.version=).pkg -target / 
+  # Password:
+  # installer: Package name is teleport-ent-(=teleport.version=)
+  # installer: Upgrading at base path /
+  # installer: The upgrade was successful.
+  $ which teleport
+  # /usr/local/bin/teleport
+  ```
 
-But first, lets see how Joe can log into the Teleport cluster. He can do this
-on his client laptop:
+</TabItem>
+<TabItem label="Linux">
 
-```code
-$ tsh --proxy=auth.example.com --insecure login --user=joe
-```
+Run the following commands to install Teleport binaries on your client system,
+including `tsh`:
 
-Note that "--user=joe" part can be omitted if `$USER` environment variable is "joe".
+(!docs/pages/includes/install-linux.mdx!)
 
-Notice that `tsh` client always needs `--proxy` flag because all client connections
-in Teleport always must to go through an SSH proxy, sometimes called an "SSH bastion".
+</TabItem>
+<TabItem label="Windows PowerShell">
 
-<Admonition
-  type="warning"
-  title="Warning"
->
-  For the purposes of this quickstart we are using the `--insecure` flag which allows
-  us to skip configuring the HTTPS/TLS certificate for Teleport proxy. Your browser will
-  throw a warning **Your connection is not private**. Click Advanced, and **Proceed to 0.0.0.0 (unsafe)**
-  to preview the Teleport UI.
+Starting with Teleport v7.2.0, most `tsh` features are supported for Windows 10
```

Comment:
At what point should we just stop mentioning this? v7 is super old by now. Can we expect anyone reading a "getting started" guide at this point to be on something newer than Teleport 7?